src/popup/pages/Popups/AccountList.vue

```diff
@@ -0,0 +1,175 @@
+    >
+      <span class="app-name">{{ appName }}</span>
+      ({{ app.host }}) {{ $t('pages.accountListConfirm.websiteRequestconnect') }}
+    </div>
+
+    <div class="permissions">
+      <template v-if="access.includes(POPUP_CONNECT_ADDRESS_PERMISSION)">
+        <span class="title">
+          <CheckMark class="icon" /> {{ $t('common.address') }}
+        </span>
+        <span class="description">
+          {{ $t('pages.accountListConfirm.addressesRequest') }}
+        </span>
+        <span class="description message">
+          {{ $t('pages.accountListConfirm.message') }}
+        </span>
+      </template>
+    </div>
+
+    <template #footer>
+      <BtnMain
+        variant="muted"
+        data-cy="deny"
+        extra-padded
+        :text="$t('pages.connectConfirm.cancelButton')"
+        @click="cancel()"
+      />
+      <BtnMain
+        data-cy="accept"
+        :text="$t('pages.connectConfirm.confirmButton')"
+        @click="confirm()"
+      />
+    </template>
+  </Modal>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, PropType } from '@vue/composition-api';
+import type {
+  IAppData,
+  IPermission,
+} from '../../../types';
+import {
+  PERMISSION_DEFAULTS,
+  POPUP_CONNECT_ADDRESS_PERMISSION,
+  POPUP_CONNECT_TRANSACTIONS_PERMISSION,
+} from '../../utils';
+import { useState } from '../../../composables/vuex';
+
+import Modal from '../../components/Modal.vue';
+import BtnMain from '../../components/buttons/BtnMain.vue';
+import TransactionInfo from '../../components/TransactionInfo.vue';
+import CheckMark from '../../../icons/check-mark.svg?vue-component';
+
+export default defineComponent({
+  components: {
+    Modal,
+    BtnMain,
+    TransactionInfo,
+    CheckMark,
+  },
+  props: {
+    app: { type: Object as PropType<IAppData>, required: true },
+    access: {
+      type: Array,
+      default: () => ([
+        POPUP_CONNECT_ADDRESS_PERMISSION,
+      ]),
+    },
+    resolve: { type: Function as PropType<() => void>, required: true },
+    // eslint-disable-next-line no-unused-vars
+    reject: { type: Function as PropType<(e: Error) => void>, required: true },
+  },
+  setup(props, { root }) {
+    const permission = useState<IPermission>('permissions', props.app.host);
+    const appName = computed(() => permission.value?.name || props.app.name);
+    const recipient = {
+      wallet: 'Superhero Wallet',
+    };
+
+    function confirm() {
+      root.$store.commit('permissions/addPermission', {
+        ...PERMISSION_DEFAULTS,
+        ...props.app,
+        ...permission.value,
+      });
+      props.resolve();
+    }
+
+    function cancel() {
+      props.reject(new Error('Rejected by user'));
+    }
+
+    return {
+      POPUP_CONNECT_ADDRESS_PERMISSION,
+      POPUP_CONNECT_TRANSACTIONS_PERMISSION,
+      appName,
+      confirm,
+      cancel,
+      recipient,
+    };
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@use '../../../styles/variables';
```

Comment:
```suggestion
@use '../../../styles/variables' as *;
```
This way there is no need to prefix all variables with `variables.`.

Reply:
Why you resolved this comment without reasoning why you won't do anything?